README.rst

```diff
@@ -12,13 +12,31 @@ Sequence Independent Molecular Replacement Based on Available Database
 
 .. image:: https://img.shields.io/badge/solution%20count-4-blue.svg?style=flat
    :target: http://www.rcsb.org/pdb/results/results.do?tabtoshow=Current&qrid=C03A7140
-   :alt: Solution count 
+   :alt: Solution count
 
-Documentation & Usage
-+++++++++++++++++++++
-* Please refer to `SIMBAD's installation page <http://simbad.readthedocs.io/en/latest/install.html>`_ for information on how to install SIMBAD and to maintain SIMBAD databases.
-* Please refer to `SIMBAD's examples page <http://simbad.readthedocs.io/en/latest/examples.html>`_ to see examples of how to run SIMBAD.
-* Please refer to `SIMBAD's documentation page <http://simbad.readthedocs.io/en/latest/contents.html>`_ for additional command line options and for API documentation.
+About
++++++
+
+SIMBAD is a sequence independant molecular replacement pipeline developed by the Rigden Lab at the University of Liverpool.
+SIMBAD provides an alternate strategy to identify search models for molecular replacement without using sequence information.
+This makes it suited to solve cases of contaminant crystallisation, and other mishaps such as mistaken identity (swapped crystallisation trays),
```

Comment:
"This makes SIMBAD suited to:
- solve cases of contaminant crystallisation, and ...
- solve unsequenced targets
- provide a sequence-independent brute-force MR approach, ...